css-size-adjust-1/Overview.bs

```diff
@@ -44,6 +45,12 @@ Introduction {#intro}
   time, this needs to be done with minimal disruption to the overall
   design of the page.
 
+  Even on a display which is large enough to prevent horizontal scrolling,
+  when a user does not zoom into the page, text size adjustment may still
+  be desirable. For example, if the viewport rules [[!CSS-DEVICE-ADAPT]]
+  describe a default zoom, text may need to be adjusted in order to be
+  readable at that default zoom level.
```

Comment:
Is this specific to zoomed-out effects (zoom < `1.0`), or is it more general than that?  (I'm not sure I entirely understand the scenario this sentence is describing.)